JsCreateString(Utf16) requires active script context?

xiaoyinl opened the pull request:
It seems to me that `JsCreateString` and `JsCreateStringUtf16` require an active script context. They both return `JsErrorNoCurrentContext` if `JsSetCurrentContext` is not called earlier. But the doc doesn't mention they "require an active script context" in the "Remarks" section. I wonder if this note should be added.

The following is the test code:

```
#include <iostream>
#include <ChakraCore.h>
#pragma comment(lib, "ChakraCore.lib")

int main(void)
{
    char s[] = "hello";
    wchar_t str[] = L"hello";
    JsValueRef val;
    JsErrorCode err;
    if ((err = JsCreateStringUtf16((const uint16_t*)str, sizeof(str), &val)) != JsNoError)
    {
        std::wcout << L"JsCreateStringUtf16 fails with error " << err << L"\n";
    }
    if ((err = JsCreateString(s, sizeof(s), &val)) != JsNoError)
    {
        std::wcout << L"JsCreateString fails with error " << err << L"\n";
    }
    getchar();
    return 0;
}
```

msftclas commented:
@xiaoyinl,
Thanks for having already signed the Contribution License Agreement. Your agreement was validated by Microsoft. We will now review your pull request.
_Thanks,
Microsoft Pull Request Bot_

dilijev commented:
@liminzhu Does this require updating the doc comments in the source code as well?

liminzhu commented on reference/JsCopyStringUtf16.md:
```diff
@@ -29,3 +29,5 @@ When start is out of range or < 0, returns JsErrorInvalidArgument
 and length combination), then `written` will be equal to 0 and call
 returns JsNoError.
+
+This API requires an active script context.
```
Thanks for pointing it out @xiaoyinl . Both API require script context. Can you change this line (and the same line for JsCreateString) to 

> Requires an active script context.

So that it matches all the other API references ([example](https://github.com/Microsoft/ChakraCore/wiki/JsCreateObject)).

xiaoyinl replied:
Done. Thank you for the review!

liminzhu commented:
@dilijev it's a good idea to change the [remark](https://github.com/Microsoft/ChakraCore/blob/master/lib/Jsrt/ChakraCore.h#L249) in header too.

xiaoyinl commented:
@liminzhu I just realized I actually updated JsCopyStringUtf16 (instead of JsCreateStringUtf16) and JsCreateString. Do JsCopyString and JsCopyStringUtf16 also require active script context? If so, I will update this patch to update all four APIs.

liminzhu commented:
No copystring actually does not require script context.

xiaoyinl commented:
@dilijev @liminzhu I updated the remarks in the header too.

xiaoyinl committed:
JsCreateString(Utf16) requires active script context

Update the doc to mention that JsCreateString and JsCreateStringUtf16 require
an active script context.

xiaoyinl commented:
I force pushed a new commit. Now it only updates two JsCreateString APIs.
